src/foam/box/HTTPReplyBox.js

```diff
@@ -37,7 +37,8 @@ foam.CLASS({
         foam.lib.formatter.JSONFObjectFormatter formatter = new foam.lib.formatter.JSONFObjectFormatter();
         // Needed because JS JSON parser doesn't support unquoted keys.
         formatter.setQuoteKeys(true);
-        formatter.setOutputShortNames(true);
+        // TODO: investigate why shortname breaks projection on the client side
```

Comment:
Can you create a ticket for this and assign to me?

Reply:
I'm looking into it.